test/check-application

```diff
@@ -1640,19 +1640,33 @@ WantedBy=multi-user.target default.target
     def testFailingPodmanService(self):
         b = self.browser
 
-        self.execute(True, "systemctl mask podman.service")
-        self.addCleanup(self.execute, True, "systemctl unmask podman.service")
-        self.execute(False, "systemctl --user mask podman.service")
-        self.addCleanup(self.execute, False, "systemctl --user unmask podman.service")
-        self.login_and_go("/podman")
-
-        # Troubleshoot action
+        try:
+            self.execute(True, "systemctl mask podman.service")
+            self.execute(False, "systemctl --user mask podman.service")
+            self.login_and_go("/podman")
+
+            # Troubleshoot action
+            b.wait_text("#app .pf-v5-c-empty-state__title", "Podman service failed")
+            b.click("#app .pf-v5-c-empty-state button")
+            b.enter_page("/system/services")
```

Comment:
This is a minor nit, but we have a `troubleshoot` button which should jump to `/system/services#/podman.socket` where we can also assert the status.

Reply:
The test above already does that, I thought we don't have to do it twice.